exasol_advanced_analytics_framework/udf_communication/peer_communicator/background_listener_interface.py

```diff
@@ -93,6 +93,11 @@ def register_peer(self, peer: Peer):
         register_message = messages.RegisterPeer(peer=peer)
         self._in_control_socket.send(serialize_message(register_message))
 
+    def send_payload(self, message: messages.Payload, payload: List[Frame]):
+        message = serialize_message(message)
```

Comment:
Do you intentionally overwrite parameter `message` here?

Reply:
Not really, will use a different variable